Fix "error: implicit declaration of function 'makeTetrix'" and another
problem fixed by the same upstream commit.
https://gitlab.com/graphviz/graphviz/-/commit/b4947d67a4ebd48ca0105d44f92e47f044e51600

Fix "error: implicitly declaring library function 'frexp'"
and "error: implicitly declaring library function 'ldexp'
https://gitlab.com/graphviz/graphviz/-/commit/da4c2ec6f24ca1b6d1752c6b5bc4389e55682147
--- cmd/lefty/dot2l/dotlex.c.orig	2016-08-09 16:02:09.000000000 -0500
+++ cmd/lefty/dot2l/dotlex.c	2020-08-27 14:52:49.000000000 -0500
@@ -252,7 +252,7 @@
     char *q;
 
     q = lexbuf;
-    if (p == '\0')
+    if (!p || *p == '\0')
         return NULL;
     while (isalnum (*p) || (*p == '_') || (!isascii (*p)))
         *q++ = *p++;
--- cmd/tools/gvgen.c.orig	2017-01-18 23:43:35.000000000 -0600
+++ cmd/tools/gvgen.c	2020-08-27 14:52:49.000000000 -0500
@@ -453,6 +453,8 @@
 	fprintf(opts.outfile, "}\ngraph {\n");
 }
 
+extern void makeTetrix(int depth, edgefn ef);
+
 int main(int argc, char *argv[])
 {
     GraphType graphType;
--- lib/sfio/sfdlen.c.orig
+++ lib/sfio/sfdlen.c
@@ -13,6 +13,8 @@
 
 #include	"sfhdr.h"
 
+#include <math.h>
+
 /*	Return the length of a double value if coded in a portable format
 **
 **	Written by Kiem-Phong Vo
--- lib/sfio/sfgetd.c.orig
+++ lib/sfio/sfgetd.c
@@ -13,6 +13,8 @@
 
 #include	"sfhdr.h"
 
+#include <math.h>
+
 /*	Read a portably coded double value
 **
 **	Written by Kiem-Phong Vo
--- lib/sfio/sfputd.c.orig
+++ lib/sfio/sfputd.c
@@ -14,6 +14,7 @@
 #include	"sfhdr.h"
 
 #include <float.h>
+#include <math.h>
 
 /*	Write out a floating point value in a portable format
 **
